projects/canopy-test-app/src/app/app.component.html

```diff
@@ -223,28 +223,47 @@
           </lg-card>
 
           <button lg-button variant="solid-primary">
-            Test solid primary button
+            Solid primary button
           </button>
           <br />
 
           <button lg-button variant="solid-secondary">
-            Test solid secondary button
+            Solid secondary button
           </button>
           <br />
 
           <button lg-button variant="outline-primary">
-            Test outline primary button
+            Outline primary button
           </button>
           <br />
 
           <button lg-button variant="outline-secondary">
-            Test outline secondary button
+            Outline secondary button
           </button>
           <br />
 
           <button lg-button variant="solid-primary" [loading]="true">
-            Test loading button
+            Loading button
           </button>
+          <br />
+
+          <button lg-button variant="solid-secondary">
+            Button with icon
+            <lg-icon [name]="'add'"></lg-icon>
```

Comment:
i think Owen mentioned a similar occurrence in that we can drop the brackets and the inner quotes

Reply:
I think we have a few of these occurrences, I'll be honest I wasn't sure what common convention was. I've swapped it on these ones. 👍